source/admin_manual/migrating_mailboxes.rst

```diff
@@ -0,0 +1,313 @@
+.. _migrating_mailboxes:
+
+===================
+Migrating mailboxes
+===================
+
+.. contents::
+   :depth: 1
+   :local:
+
+General
```

Comment:
Can we have this warning here? Or something similar:

WARNING: Badly done migration will cause your IMAP and/or POP3 clients to re-download all mails. Read this page carefully.